tests/ingresstests/ingressclass_test.go

```diff
@@ -179,10 +187,16 @@ func TestDefaultIngressClassChange(t *testing.T) {
 	// ingress status IP comes back
 	g := gomega.NewGomegaWithT(t)
 
-	ingClassName, ingressName, ns := "avi-lb", "foo-with-class2", "default"
-	modelName := "admin/cluster--Shared-L7-1"
+	ingClassName := objNameMap.GenerateName("avi-lb")
+	ingressName := objNameMap.GenerateName("foo-with-class")
+	ns := integrationtest.DEFAULT_NAMESPACE
+	modelName := MODEL_NAME_PREFIX + "1"
+	svcName := objNameMap.GenerateName("avisvc")
 
-	SetUpTestForIngress(t, modelName)
+	time.Sleep(time.Second * 5)
+	ingestionQueue := utils.SharedWorkQueue().GetQueueByName(utils.ObjectIngestionLayer)
+	ingestionQueue.SyncFunc = syncFromIngestionLayerWrapper
```

Comment:
the syncfunc is updated in each UT to allow UTs to be run individually instead of a single suite always